client/my-sites/stats/stats-subscribers-overview/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { CountComparisonCard } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import React from 'react';
+
+function SubscribersOverviewCardStats() {
+	const overviewCardStats = [
+		{
+			heading: translate( 'Today' ),
+			count: 1410,
+		},
+		{
+			heading: translate( '30 days ago' ),
+			count: 1080,
+			previousCount: 1080,
+		},
+		{
+			heading: translate( '60 days ago' ),
+			count: 1000,
+			previousCount: 1080,
+		},
+		{
+			heading: translate( '90 days ago' ),
+			count: 970,
+			previousCount: 1080,
+		},
+	];
+	return overviewCardStats;
+}
+
+function SubscribersOverviewCards() {
+	const overviewCardStats = SubscribersOverviewCardStats();
+
+	return (
+		<div className="overview-cards-list">
```

Comment:
All the cards are stacked one after the other, and they take full width so you probably forgot to include a CSS file in the PR.

Reply:
I imagine the styling will come in a follow-up PR?